syntax = "proto3";

package perception;

message PerceptionOptions { double frequency = 1; }
